README.md

```diff
@@ -1,70 +1,20 @@
-# Getting Started with Create React App
+# Crane Cloud CMS Frontend
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is a repo for the CMS Frontend for Crane Cloud.
 
-## Available Scripts
+## Setup
 
-In the project directory, you can run:
+Follow these steps to have a local running copy of the CMS Frontend.
 
-### `yarn start`
+To set up the Frontend, clone the repo:
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+1 - Clone the repo https://github.com/crane-cloud/cms-frontend.git If `master` is not up to date, checkout `develop`.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+2 - We prefer `Yarn` over NPM but both are supported.
 
-### `yarn test`
+3 - Install the dependencies using 'yarn' or 'npm install' with ```yarn install``` or ```npm install```
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+4 - Create a `.env` file and add the CMS backend URL.
```

Comment:
Hey @neelxie, I think you might need to specify the keys to be added to the `.env` file something like; `URL_BACKEND`

Reply:
@Mubangizi this has been fixed sir.